pkg/fluxApplication/FluxApplicationService.go

```diff
@@ -0,0 +1,126 @@
+package fluxApplication
+
+import (
+	"context"
+	"github.com/devtron-labs/common-lib/utils/k8s"
+	"github.com/devtron-labs/devtron/api/helm-app/gRPC"
+	"github.com/devtron-labs/devtron/pkg/cluster"
+	"github.com/devtron-labs/devtron/pkg/fluxApplication/bean"
+	"go.uber.org/zap"
+)
+
+type FluxApplicationService interface {
+	ListApplications(ctx context.Context, clusterIds []int) ([]*bean.FluxApplicationListDto, error)
+	ConvertClusterBeanToClusterConfig(clusters []*cluster.ClusterBean) ([]*gRPC.ClusterConfig, error)
```

Comment:
why are we binding this method to this interface since it's not used by any other external interfaces